### PR TITLE
adding traditional version to almost.md

### PR DESCRIPTION
**Pull request by jopham**

### CODE REVIEW

**Comment by jopham** [comment]
@rayjcwu

File: grammar/almost.md [hunk 1]
```diff
@@ -2,37 +2,45 @@
 
 
 
-**差一点儿 | chàyīdiǎnr**
+**差一点儿 | 差一點兒 | chàyīdiǎnr**
```
Comment (rayjcwu) [inline grammar/almost.md]: it is correct but I seldom saw people not from China saying 兒

File: grammar/almost.md [hunk 2]
```diff
@@ -2,37 +2,45 @@
...
 
 **Examples:**
 * 我差点儿闹笑话了。  
+我差點兒笑話了。
```
Comment (rayjcwu) [inline grammar/almost.md]: same

File: grammar/almost.md [hunk 3]
```diff
@@ -2,37 +2,45 @@
...
+**差一点儿 | 差一點兒 | chàyīdiǎnr**
 
 **Examples:**
 * 我差点儿闹笑话了。  
+我差點兒笑話了。  
 Wǒ chàdiǎnr nào xiàohuà le.  
 I almost made a fool of myself.
 
 * 我差点儿把寿司加热。  
+我差點兒把壽司加熱。
```
Comment (rayjcwu) [inline grammar/almost.md]: same

File: grammar/almost.md [hunk 4]
```diff
@@ -2,37 +2,45 @@
...
 Wǒ chàdiǎnr nào xiàohuà le.  
 I almost made a fool of myself.
 
 * 我差点儿把寿司加热。  
+我差點兒把壽司加熱。  
 Wǒ chàdiǎnr bǎ shòusī jiārè.  
 I almost heated the sushi.
 
 * 我差点儿走错地方。  
+我差點兒走錯地方。
```
Comment (rayjcwu) [inline grammar/almost.md]: same

File: grammar/almost.md [hunk 5]
```diff
@@ -2,37 +2,45 @@
...
 Wǒ chàdiǎnr zǒucuò dìfang.  
 I almost went to the wrong place.
 
 * 我差点儿被车撞了。  
+我差點兒被車撞了。  
 Wǒ chàdiǎnr bèi chē zhuàng le.  
 I almost got hit by a car.
 
 * 我上次看见一个老朋友，很激动，差点儿摔倒了。  
+我上次看見一個老朋友，很激動，差點兒摔倒了。
```
Comment (rayjcwu) [inline grammar/almost.md]: why you will fell down when you are excited?

File: grammar/almost.md [hunk 6]
```diff
@@ -2,37 +2,45 @@
...
 Wǒ chàdiǎnr bǎ shòusī jiārè.  
 I almost heated the sushi.
 
 * 我差点儿走错地方。  
+我差點兒走錯地方。  
 Wǒ chàdiǎnr zǒucuò dìfang.  
 I almost went to the wrong place.
 
 * 我差点儿被车撞了。  
+我差點兒被車撞了。
```
Comment (rayjcwu) [inline grammar/almost.md]: same

File: grammar/almost.md [hunk 7]
```diff
@@ -2,37 +2,45 @@
...
 Wǒ chàdiǎnr bèi chē zhuàng le.  
 I almost got hit by a car.
 
 * 我上次看见一个老朋友，很激动，差点儿摔倒了。  
+我上次看見一個老朋友，很激動，差點兒摔倒了。  
 Wǒ shàngcì kànjian yī ge lǎopéngyou，hěn jīdòng，chàdiǎnr shuāidǎo le.  
 Last time I saw my old friend, I got very excited and almost fell down.
 
 * 我差点儿认错人了。  
+我差點兒認錯人了。
```
Comment (rayjcwu) [inline grammar/almost.md]: same

File: grammar/almost.md [hunk 8]
```diff
@@ -2,37 +2,45 @@
...
 Wǒ shàngcì kànjian yī ge lǎopéngyou，hěn jīdòng，chàdiǎnr shuāidǎo le.  
 Last time I saw my old friend, I got very excited and almost fell down.
 
 * 我差点儿认错人了。  
+我差點兒認錯人了。  
 Wǒ chàdiǎnr rèncuò rén le.  
 I almost mistook that person (for another person).
 
 * 我差点儿没赶上飞机。  
+我差點兒沒趕上飛機。
```
Comment (rayjcwu) [inline grammar/almost.md]: same

File: grammar/almost.md [hunk 9]
```diff
@@ -2,37 +2,45 @@
...
 Wǒ chàdiǎnr rèncuò rén le.  
 I almost mistook that person (for another person).
 
 * 我差点儿没赶上飞机。  
+我差點兒沒趕上飛機。  
 Wǒ chàdiǎnr méi gǎnshàng fēijī.  
 I almost missed my plane.
 
 * 我差点儿没认出你。  
+我差點兒沒認出妳。
```
Comment (rayjcwu) [inline grammar/almost.md]: I would say 差點認不出你

**Comment by rayjcwu** [comment]
@jopham

**Pull request merged**